tests/end-to-end/director/oauth20_test.go

```diff
@@ -190,3 +292,83 @@ func TestDeleteSystemAuthFromIntegrationSystem(t *testing.T) {
 	intSysAfterDelete := getIntegrationSystem(t, ctx, intSys.ID)
 	require.Empty(t, intSysAfterDelete.Auths)
 }
+
+func TestDeleteSystemAuthFromIntegrationSystemUsingApplicationMutationShouldReportError(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	name := "int-system"
+
+	t.Log("Create integration system")
+	intSys := createIntegrationSystem(t, ctx, name)
+	require.NotEmpty(t, intSys)
+	defer deleteIntegrationSystem(t, ctx, intSys.ID)
+
+	intSysAuth := generateClientCredentialsForIntegrationSystem(t, ctx, intSys.ID)
+	require.NotEmpty(t, intSysAuth)
+	defer deleteClientCredentialsForIntegrationSystem(t, intSysAuth.ID)
+
+	deleteSystemAuthForApplicationRequest := fixDeleteSystemAuthForApplication(intSysAuth.ID)
+	deleteOutput := graphql.SystemAuth{}
+
+	// WHEN
+	t.Log("Delete system auth for integration system using application mutation")
+	err := tc.RunOperation(ctx, deleteSystemAuthForApplicationRequest, &deleteOutput)
+
+	//THEN
+	require.Error(t, err)
+}
+
+func TestDeleteSystemAuthFromIntegrationSystemUsingRuntimeMutationShouldReportError(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	name := "int-system"
+
+	t.Log("Create integration system")
+	intSys := createIntegrationSystem(t, ctx, name)
+	require.NotEmpty(t, intSys)
+	defer deleteIntegrationSystem(t, ctx, intSys.ID)
+
+	intSysAuth := generateClientCredentialsForIntegrationSystem(t, ctx, intSys.ID)
+	require.NotEmpty(t, intSysAuth)
+	defer deleteClientCredentialsForIntegrationSystem(t, intSysAuth.ID)
+
+	deleteSystemAuthForRuntimeRequest := fixDeleteSystemAuthForRuntime(intSysAuth.ID)
+	deleteOutput := graphql.SystemAuth{}
+
+	// WHEN
+	t.Log("Delete system auth for integration system using runtime mutation")
+	err := tc.RunOperation(ctx, deleteSystemAuthForRuntimeRequest, &deleteOutput)
+
+	//THEN
+	require.Error(t, err)
+}
+
+func deleteClientCredentialsForApplication(t *testing.T, authID string) {
```

Comment:
What do you think about moving all those  to the `fixtures_queries_test.go`?

Reply:
If this is the convention, previously I saw that such methods are defined within its domain, which I prefer.